src/components/Editor/Tabs.css

```diff
@@ -3,7 +3,6 @@
   width: 100%;
   height: 30px;
   display: flex;
-  justify-content: space-between;
```

Comment:
This doesn't affect horizontal mode.
![horizontal](https://cloud.githubusercontent.com/assets/1755089/23468783/be06a988-fec7-11e6-96b4-c355d2ed1f25.png)